app/config.py

```diff
@@ -39,6 +46,13 @@ class Config(metaclass=MetaFlaskEnv):
     HTTP_SCHEME = "https"
     FRONTEND_HOSTNAME = None
 
+    REDIS_URL = os.getenv("REDIS_URL")
+    REDIS_ENABLED = REDIS_URL is not None
+
+    # 100 requests every 5 minutes
+    DOCUMENT_AUTHENTICATION_RATE_LIMIT = int(os.getenv("DOCUMENT_AUTHENTICATION_RATE_LIMIT", "100"))
+    DOCUMENT_AUTHENTICATE_RATE_INTERVAL = int(os.getenv("DOCUMENT_AUTHENTICATE_RATE_INTERVAL", "300"))
```

Comment:
Optional: Could suffix this name with `_SECONDS` for added clarity

Reply:
Done 👍